modules/gdscript/gdscript_functions.cpp

```diff
@@ -63,6 +63,8 @@ const char *GDScriptFunctions::get_func_name(Function p_func) {
 		"ceil",
 		"round",
 		"abs",
+		"abs",
```

Comment:
Could the bug mentioned by @aaronfranke be there because of this repitition in name?